src/widgets/wxWidgetsBasicUI.cpp

```diff
@@ -0,0 +1,236 @@
+/*!********************************************************************
+
+Audacity: A Digital Audio Editor
+
+@file wxWidgetsBasicUI.cpp
+
+Paul Licameli
+
+**********************************************************************/
+#include "wxWidgetsBasicUI.h"
+#include "MemoryX.h" // for Destroy_ptr
+#include "widgets/ErrorDialog.h"
+#ifdef HAS_SENTRY_REPORTING
+#include "widgets/ErrorReportDialog.h"
+#endif
+#include "widgets/AudacityMessageBox.h"
+#include "ProgressDialog.h"
+#include <wx/app.h>
+#include <wx/progdlg.h>
+#include <wx/windowptr.h>
+
+using namespace BasicUI;
+
+wxWidgetsWindowPlacement::~wxWidgetsWindowPlacement() = default;
+
+wxWidgetsBasicUI::~wxWidgetsBasicUI() = default;
+
+void wxWidgetsBasicUI::DoCallAfter(const Action &action)
+{
+   wxTheApp->CallAfter(action);
+}
+
+void wxWidgetsBasicUI::DoYield()
+{
+   wxTheApp->Yield();
+}
+
+namespace {
+wxWindow *GetParent(const BasicUI::WindowPlacement &placement)
```

Comment:
GetWindow?

Reply:
Maybe, it's just a name used only within this one .cpp file in a few places.

It is always getting a Window pointer for the purposes of parenting another new window.  Never for more general purpose.